cpp/unreal_plugins/CoreUtils/Source/CoreUtils/Log.h

```diff
@@ -0,0 +1,52 @@
+//
+// Copyright(c) 2022 Intel. Licensed under the MIT License <http://opensource.org/licenses/MIT>.
+//
+
+#pragma once
+
+#include <string>
+
+#include <filesystem>
+#include <utility>
```

Comment:
Combine these and sort alphabetically since they all are part of standard library?

Reply:
Ah yes of course. Good catch. I used to be including <string.h> which I considered to be part of the lower-level C standard library, but now that I'm using <string> it should be alphabetical with the rest of the C++ standard library.